test/js/specs/unit/collections/AccessPolicy.spec.js

```diff
@@ -0,0 +1,1519 @@
+        state.policy.add(
+          makeRule({
+            subject: "alice",
+            changePermission: true
+          })
+        );
+        expect(state.policy.hasOwner()).to.be.true;
+      });
+
+      it("hasOwner returns false when no changePermission rule exists", function() {
+        state.policy.add(
+          makeRule({
+            subject: "bob",
+            read: true
+          })
+        );
+        expect(state.policy.hasOwner()).to.be.false;
+      });
+
+      it("reflects the new rights holder after replaceRightsHolder()", () => {
+        state.policy.add(
+          makeRule({
+            subject: "uid:new",
+            changePermission: true
+          })
+        );
+        const obj = new DataONEObject({
+          id: "urn:uuid:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+          rightsHolder: "alice"
+        });
+        state.policy.dataONEObject = obj;
+
+        expect(state.policy.dataONEObject.rightsHolder).to.not.be.null;
+
+        state.policy.replaceRightsHolder("uid:new");
+        const actual = state.policy.dataONEObject.get("rightsHolder");
+        expect(actual).to.equal("uid:new");
+      });
+
+      it("replaceRightsHolder swaps the changePermission rule into the object rightsHolder", function() {
+        const obj = new DataONEObject({
+          rightsHolder: null,
+          systemMetadata: { rightsHolder: null }
+        });
+        state.policy.dataONEObject = obj;
+        state.dataONEObject = obj;
+
+        const ownerRule = makeRule({
+          subject: "newOwner@example.org",
+          changePermission: true
+        });
+        state.policy.add(ownerRule);
+        state.policy.replaceRightsHolder();
+
+        expect(state.dataONEObject.get("rightsHolder")).to.equal(
+          "newOwner@example.org"
+        );
+        // The rule should now be removed from the collection.
+        expect(state.policy.findWhere({ changePermission: true })).to.be.undefined;
+      });
+
+      it("replaceRightsHolder does nothing when there is no owner rule", function() {
+        state.policy.add(
+          makeRule({ subject: "bob", read: true })
+        );
+        const originalRightsHolder = state.dataONEObject.get("rightsHolder");
+        state.policy.replaceRightsHolder();
+        expect(state.dataONEObject.get("rightsHolder")).to.equal(originalRightsHolder);
+        expect(state.policy.length).to.equal(1); // collection unchanged
+      });
+    });
+
+    //-----------------------------------------------------------------------
+    // 3.8  serialize()
+    //-----------------------------------------------------------------------
+    describe("serialize()", function() {
+      it.skip("EXPECTED FAILURE: always returns an <accesspolicy> element (never null)", function() {
+        const el = state.policy.serialize();
+        expect(el && el.nodeType).to.equal(1);
+        expect(el.tagName.toLowerCase()).to.equal("accesspolicy");
+      });
```

Comment:
Returning null when a node is empty is the general serialize pattern that MetacatUI uses. We use `AccessPolicy.serialize` in `DataONEObject.serializeSysMeta`:

https://github.com/NCEAS/metacatui/blob/main/src/js/models/DataONEObject.js#L1129-L1149

We don't want to add an empty accessPolicy to the sysMeta. However, is there a reason you're thinking of that makes it better to return `<accesspolicy></accesspolicy>`?